pkg/apis/servicecatalog/checksum/unversioned/checksum.go

```diff
@@ -37,7 +37,12 @@ func InstanceSpecChecksum(spec servicecatalog.InstanceSpec) string {
 	specString += fmt.Sprintf("planName: %v\n", spec.PlanName)
 
 	if spec.Parameters != nil {
-		specString += fmt.Sprintf("parameters:\n\n%v\n\n", string(spec.Parameters.Raw))
+		params := spec.Parameters
+		if params.SecretRef != nil {
+			specString += fmt.Sprintf("parameters.secretRef: %v\n", params.SecretRef.Name)
```

Comment:
If this really is ok, can we add a comment to that effect here so the next person looking at the code doesn't think its a design flaw, rather its done on purpose?